components/calendar/src/ethiopic.rs

```diff
@@ -207,16 +205,19 @@ impl Date<Ethiopic> {
             return Err(DateTimeError::OutOfRange);
         }
 
+        Ok(Date::from_raw(EthiopicDateInner(inner, false), Ethiopic))
+    }
+
+    pub fn set_amete_alem(&self, value: bool) -> Result<Date<Ethiopic>, DateTimeError> {
```

Comment:
no, this should just be `fn set_amete_alem(&mut self, value: bool) { self.inner.set_amete_alem(value)  }`